cli/index.js

```diff
@@ -1,13 +1,16 @@
 var program = require('commander'),
     chalk = require('chalk'),
     wake = require('../'),
+    relay = require('../').relay,
```

Comment:
we don't need two requires here. why not just `require('wake').relay`? also i think, that we could change api to use it like `wake({ relay })`

Reply:
Ok